support-payment-api/src/main/scala/conf/AcquisitionsStremServiceConfigLoader.scala

```diff
@@ -0,0 +1,34 @@
+package conf
```

Comment:
typo in the filename!

Reply:
ahhhh, not even in `Acquisitions` either!